operator/internal/manifests/build_test.go

```diff
@@ -263,7 +263,7 @@ func TestBuildAll_WithFeatureFlags_EnableTLSServiceMonitorConfig(t *testing.T) {
 			continue
 		}
 
-		secretName := fmt.Sprintf("%s-http-metrics", name)
+		secretName := fmt.Sprintf("%s-http-tls", name)
```

Comment:
Watch out, my PR #6224 also addresses this issue by removing `-metrics` entirely. FYI after re-working my PR a couple of times I err'd on naming the services `lokistack-dev-component-http` and `lokistack-dev-component-grpc`. You have two choices either you undo this renames in your PR or someone of us resolves the merge conflict. ;)

Reply:
I can rollback my changes, but I do believe `tlsMetricsSercetVolume = "tls-metrics-secret"` should be `tlsSercetVolume = "tls-secret"` since we use it here for both metrics and main gateway. What do you think?

Reply:
@aminesnow If #6224 is missing to rename the volume, then we should keep the name change to `tls-secret` in this PR.

Reply:
Let's keep it as is the conflict is very limited for both of us.